apps/integration_tests/selenium_cases.py

```diff
@@ -23,6 +23,7 @@ class Action(Enum):
     GET_SAMPLE_TOKEN_PKCE_START = 11
     SLEEP = 12
     VALIDATE_EMAIL_NOTIFICATION = 13
+    CHECK_DATE_FORMAT = 14

```

Comment:
Newbie question: How do you run the integration tests? is it the same as in the readme, something like python runtests.py apps.integration_tests ?

Reply:
No, the Selenium tests run in their own container (not the web container) so to run these I modified the run_selenium_tests_remote.sh script to have this line:
export TESTS_LIST="./apps/integration_tests/selenium_spanish_tests.py"
And then ran the script against a test instance.

Reply:
Oh! Took a look at run_selenium_tests_remote.sh and the doc on the canary instance https://confluence.cms.gov/display/BB2/Deploying+the+API this makes more sense now. Thank you!